pkgs/by-name/sp/spiel/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  stdenv,
+  python312Packages,
```

```suggestion
  python3Packages,
```
Best to unpin the python version and fix it in the future if it ever breaks.